src/test/system_tests/test_ceph_s3.js

```diff
@@ -76,309 +33,13 @@ let stats = {
 
 let tests_list;
 
+const s3_tests_black_list_url = 'src/test/system_tests/s3-tests-lists/s3_tests_black_list.txt';
```

Comment:
it's better to pass the black lists as an arg to the script, so we can use different lists for different setups. 
you can send an arg `--ignore_lists path/to/list1,/path/to/list2`. in the test just split it
```javascript
black_lists = argv.ignore_lists.split(',');
```